packages/fxa-content-server/scripts/test-ci-remote.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash -ex
+
+DIR=$(dirname "$0")
+cd "$DIR/.."
+echo -e "\n###################################"
+echo "# testing fxa-content-server"
+echo -e "###################################\n"
+
+function test_suite() {
+  local suite=$1
+  node tests/intern.js \
+    --suites="${suite}" \
+    --fxaAuthRoot=https://api-accounts.stage.mozaws.net/v1 \
+    --fxaContentRoot=https://accounts.stage.mozaws.net/ \
+    --fxaOAuthApp=https://123done-stage.dev.lcip.org/ \
+    --fxaUntrustedOauthApp=https://321done-stage.dev.lcip.org/ \
+    --fxaEmailRoot=http://restmail.net \
+    --fxaProduction=true \
+    --output="../../artifacts/tests/${suite}-${numGroups}-${i}-results.xml" \
+    --firefoxBinary=./firefox/firefox \
+    || \
+  node tests/intern.js \
+    --suites="${suite}" \
+    --fxaAuthRoot=https://api-accounts.stage.mozaws.net/v1 \
+    --fxaContentRoot=https://accounts.stage.mozaws.net/ \
+    --fxaOAuthApp=https://123done-stage.dev.lcip.org/ \
+    --fxaUntrustedOauthApp=https://321done-stage.dev.lcip.org/ \
+    --fxaEmailRoot=http://restmail.net \
+    --fxaProduction=true \
+    --output="../../artifacts/tests/${suite}-${numGroups}-${i}-results.xml" \
+    --firefoxBinary=./firefox/firefox \
+    --grep="$(<rerun.txt)"
+}
+
+yarn lint
+
+cd ../../
```

Comment:
Also TODO. Curl `__version__` to get the correct version of the tests to be run (i.e., running the `main` branch for tests when stage is on v1.nnn.m will result in false negatives)